js/app/src/traffic_stats/Dashboard.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+	import RequestsBar from "./RequestsBar.svelte";
+	import Queue from "./Queue.svelte";
+	import type { ActivityLog, TaskStatus, RequestBreakdown } from "./utils";
+
+	import { Block, BlockTitle } from "@gradio/atoms";
+	import Row from "../components/Row/Row.svelte";
+	import Column from "../components/Column/Column.svelte";
+	import Form from "../components/Form/Form.svelte";
+	import Accordion from "../components/Accordion/Accordion.svelte";
+
+	const total_requests_fn = (breakdown: RequestBreakdown) => {
+		return Object.values(breakdown).reduce((a, b) => a + b, 0);
+	};
+	export let sessions: ActivityLog["sessions"];
+	export let request_breakdown: ActivityLog["request_breakdown"];
+	export let requests_per_fn: ActivityLog["requests_per_fn"];
+	export let event_count_per_stage: ActivityLog["event_count_per_stage"];
+	export let queue_preview: ActivityLog["queue_preview"] | undefined =
+		undefined;
+	export let active_workers: ActivityLog["active_workers"];
+</script>
+
+<Column>
+	<Row>
+		<Form>
+			<Block>
+				<BlockTitle>Total Sessions</BlockTitle>
+				<div class="stat">{sessions}</div>
+			</Block>
+			<Block>
+				<BlockTitle>Total Requests</BlockTitle>
+				<div class="stat">
+					{total_requests_fn(request_breakdown)}
+				</div>
+			</Block>
+		</Form>
+	</Row>
+	<Block>
+		<BlockTitle info="Hover for more details.">Request Breakdown</BlockTitle>
+		<div class="requests-breakdown">
+			<RequestsBar
+				{request_breakdown}
+				total_requests={total_requests_fn(request_breakdown)}
+			/>
+		</div>
+		<Accordion open={false} label="Requests per Function">
+			<div class="fn-breakdowns">
+				{#each requests_per_fn as { fn, duration, request_breakdown }}
+					{#if total_requests_fn(request_breakdown) > 0}
+						<div class="fn-breakdown">
+							<div class="fn">
+								<span>{fn}</span>
+								<span class="details"
+									>{total_requests_fn(request_breakdown)} requests &bull; {duration.toFixed(
+										1
+									)}s of processing per success</span
+								>
```

Comment:
Something like
```suggestion
								<span class="details">
									{total_requests_fn(request_breakdown)} requests
									&bull;
									{duration.toFixed(1)}s of processing per success
								</span>
```
would be more readable.